pkg/resources/factory.go

```diff
@@ -49,16 +50,70 @@ func (rf *resourceFactory) GetResourceServer(rp types.ResourcePool) (types.Resou
 }
 
 // GetResourcePool returns and instance of ResourcePool for a ResourceConfig
-func (rf *resourceFactory) GetResourcePool(rc *types.ResourceConfig) types.ResourcePool {
-	glog.Infof("Resource pool type: %s", rc.DeviceType)
-	switch rc.DeviceType {
-	case "vfio":
-		return newVfioResourcePool(rc)
+func (rf *resourceFactory) GetInfoProvider(name string) types.DeviceInfoProvider {
```

Comment:
GetResourcePool -> GetInfoProvider in the comment

Reply:
fixed